doc/getting_started/index.rst

```diff
@@ -246,6 +246,36 @@ directory using west:
          cd zephyrproject
          west update
 
+.. rst-class:: numbered-step
+
+Export Zephyr CMake package
+***************************
+
+Exporting Zephyr as a :ref:`cmake_pkg` makes it possible for CMake to automatically find and load
```

Comment:
But the introduction already states:
```
The Zephyr CMake package ensures that CMake can automatically select a Zephyr
to use for building the application, whether it is a Zephyr repository application, 
Zephyr workspace application, or a Zephyr freestanding application.

When developing a Zephyr-based application, then a developer simply needs 
to write find_package(Zephyr) in the beginning of the application
 :file:`CMakeLists.txt` file.

To use the Zephyr CMake package it must first be exported to the CMake user 
package registry. This is means creating a reference to the current Zephyr 
installation inside the CMake user package registry.
```

isn't that explanation enough why you want to export Zephyr to CMake ?
And then the following sections are how to do the actual export, and I don't think we should repeat the introduction there:  [Zephyr CMake Package](https://github.com/tejlmand/zephyr/blob/zephyr_base_improvement/doc/guides/zephyr_cmake_package.rst#zephyr-cmake-package)

Reply:
Yes, it's enough, however:
1. My hunch is that many people won't click the link, and as long as this text looks vaguely optional, they will not call `west zephyr-export` in case it has unwanted side effects.
2. For the vast majority, the takeaway from the linked document should be "call `west zephyr-export`", especially since this is a getting started guide (right? correct me if I'm wrong :D). The less they need to read before they do that, the better (even though the document is easy to read).
3. I think people want clear instructions in a getting started guide: "Do x [if y] [unless z]"

I'm not suggesting you duplicate more than a sentence or two and I think the benefit is very large for new (possibly overwhelmed) users.

Reply:
>  especially since this is a getting started guide

No, the getting started guide is here:
[Export Zephyr CMake package](https://github.com/tejlmand/zephyr/blob/zephyr_base_improvement/doc/getting_started/index.rst#export-zephyr-cmake-package)

```
Exporting Zephyr as a :ref:`cmake_pkg` makes it possible for CMake to
automatically find and load boilerplate code for building a Zephyr application.

Zephyr CMake package is exported with the following command
....
```

This is the additional guide, that explains the feature in more details.

Reply:
> No, the getting started guide is here:

Isn't that where we are commenting?

    doc/getting_started/index.rst